app/components/landing_page_search_bar_component.html.erb

```diff
@@ -0,0 +1,18 @@
+<%= form_with url: @url, local: true, method: @method, class: @classes.join(' '), scope: @prefix, role: 'search', **@form_options do |f| %>
+  <%= render Blacklight::HiddenSearchStateComponent.new(params: @params) %>
+
+  <%= f.label @query_param, scoped_t('search.label'), class: 'sr-only visually-hidden' %>
+  
+  <div class="row">
+    <div class="input-group col-12">
+      <%= f.search_field @query_param, value: @q, placeholder: scoped_t('search.placeholder'), class: "search-q q form-control rounded-0 me-xl-3", autofocus: @autofocus, aria: { label: scoped_t('search.label') }  %>
+    </div>
+    <div class="search-links col-12">
+      <ul class="ps-md-0 pt-3">
+        <li><%= link_to 'Advanced Search', advanced_search_catalog_path %></li>
+        <li><%= link_to 'Browse all collections', '/collections' %></li>
+        <li><%= link_to 'Browse respositories', '/repositories' %></li>
```

Comment:
```suggestion
        <li><%= link_to 'Browse repositories', helpers.arclight_engine.repositories_path %></li>
```

Reply:
The front-page spelling mistake... Thanks for catching that.

Thanks for the path helper too. I tried a bunch of combos and forgot to come back to it. I was missing the underscore.